redbot/cogs/trivia/session.py

```diff
@@ -282,41 +282,47 @@ def force_stop(self):
         LOG.debug("Force stopping trivia session; #%s in %s", channel, channel.guild.id)
 
     async def pay_winner(self, multiplier: float):
```

Comment:
Since this is not a public API, it would be clearer if this got renamed to `pay_winners()` and the usage of it got updated appropriately.
```suggestion
    async def pay_winners(self, multiplier: float):
```